### PR TITLE
List insertions

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Waiting for [#269](https://github.com/gtk-rs/gtk/pull/269) to be merged.

### CODE REVIEW

File: src/process_viewer.rs [hunk 1]
```diff
@@ -180,11 +180,12 @@ fn create_and_fill_model(list_store: &mut gtk::ListStore, pid: i64, cmdline: &st
...
     let val2 = memory.to_value();
-    let top_level = list_store.append();
-    list_store.set_value(&top_level, 0, &val1);
-    list_store.set_value(&top_level, 1, &name.to_value());
-    list_store.set_value(&top_level, 2, &format!("{:.1}", cpu).to_value());
-    list_store.set_value(&top_level, 3, &val2);
+    list_store.insert_with_values(None,
+                                  &[0, 1, 2, 3],
+                                  &[&val1,
+                                    &name.to_value(),
```
Comment (gkoz) [inline src/process_viewer.rs]: you should not need `to_value`

File: src/process_viewer.rs [hunk 2]
```diff
@@ -385,7 +370,19 @@ fn main() {
     let m_display_tab = Rc::new(RefCell::new(display_tab));
 
     gtk::timeout_add(1500, move || {
+        // first part, deactivate sorting
+        let model = procs.left_tree.get_model().unwrap()
+                                   .downcast::<TreeSortable>().unwrap();
```
Comment (gkoz) [inline src/process_viewer.rs]: `let model = procs.list_store`?

Comment (GuillaumeGomez) [inline src/process_viewer.rs]: If it's too easy, where's the fun? :p

Comment (gkoz) [inline src/process_viewer.rs]: Actually I don't think any upcasts and downcasts in this crate are necessary.

**Pull request merged**